{{ cookiecutter.project_slug }}/pyproject.toml

```diff
@@ -27,10 +27,12 @@ requires-python = "~=3.9"
 dynamic = ["version"]
 
 dependencies = [
-    "oteapi-core >=0.5.2,<1",
-    "pydantic >=1.10.12,<2",
+    "oteapi-core >=0.6.0,<1",
```

Comment:
The way dependencies are set is very confusing. Why can't we set pydantic here instead of in the additional_dependencies?

Reply:
As discussed offline, these are independent things. The pydantic dependency listed in the `.pre-commit-config.yaml` file for the `mypy` hook is just that: An additional dependency for the `mypy` hook. This is in conjunction with the usage of the `mypy` plugin in `pydantic` (see the change to the `tool.mypy` block of the `pyproject.toml` file).

Here, `pydantic` is removed in order to rely on `oteapi-core` to determine the correct `pydantic` version to use. As this is a plugin repository generator for OTEAPI Core, I find it better to let OTEAPI Core determine as many things as possible.